tests/ui/useless_attribute.fixed

```diff
@@ -1,6 +1,6 @@
 //@aux-build:proc_macro_derive.rs
 
-#![allow(unused)]
+#![allow(unused, clippy::duplicated_attributes)]
```

Comment:
Why allow? What's the duplicate here?

Reply:
the duplicated attribute was introduced in https://github.com/rust-lang/rust-clippy/commit/81f5969704b57215ac12a78459c8ccfbad9be654 - it seems better to change the attribute to still test the same thing without piling on suppressions.